navigation/navigation-style.css

```diff
@@ -0,0 +1,107 @@
+/* topic-name-slide  */
```

Comment:
Please remove this file.
Just use slide from !!template!! file.
If there are no slide that you need.. please add it.
If there are any issues please fix them ;)

Reply:
done